packages/safety-suite-api/src/config.ts

```diff
@@ -158,19 +158,36 @@ export interface BasicConfig extends NestedConfiguration {
       };
       strategy: string;
     };
-    performanceDashboard: {
-      accidents: {enabled: boolean};
-      apiTarget: string;
-      developmentPlans: {enabled: boolean};
-      enabled: boolean;
-      enforcements: {enabled: boolean};
-      expiringDocuments: {enabled: boolean};
-      fallbackFrontendUrl: string;
-      frontendUrl: string;
-      hierarchyFilter: {enabled: boolean};
-      injuries: {enabled: boolean};
-      newHotness: {enabled: boolean};
-      watchlist: {enabled: boolean};
+    dashboards: {
```

Comment:
You have to also update the `exampleConfig` below in order to support the type guard. Also this will break any uses of `performanceDashboard`, so this will need a `major` version bump.